Basic/DistributedAuthoritySocialHub/Assets/Scripts/UI/IngameUI/TextChatManager.cs

```diff
@@ -63,9 +63,41 @@ void Start()
             m_Messages.Add(new ChatMessage("Sample Devs", "Hey, we hope you enjoy our sample :)"));
         }
 
-        void OnDestroy()
+        private void OnTextEnter(KeyDownEvent evt)
+        {
+            if (evt.keyCode is KeyCode.Return or KeyCode.KeypadEnter)
+            {
+                SendMessage();
+                m_MessageInputField.schedule.Execute(() => m_MessageInputField.Focus()).ExecuteLater(10);
```

Comment:
Can make the delay a constant.
And the two privates in this class.

Reply:
done